lib/css/CssParser.js

```diff
@@ -755,7 +768,14 @@ class CssParser extends Parser {
 
 						break;
 					}
-					case CSS_MODE_AT_IMPORT_INVALID:
+					case CSS_MODE_AT_IMPORT_INVALID: {
+						if (importData) {
+							const dep = new ConstDependency("", [importData.start, end]);
+							module.addPresentationalDependency(dep);
+						}
+						scope = CSS_MODE_TOP_LEVEL;
+						break;
+					}
```

Comment:
Why we need it?

Reply:
Because eslint has [no-fallthrough](https://eslint.org/docs/latest/rules/no-fallthrough) rule